addons/l10n_ma/data/template/account.account-ma.csv

```diff
@@ -1,619 +1,632 @@
+"pcg_1356","1356","Provisions for Acquisition and Construction of Housing","equity","False","Provisions pour acquisition et construction de logements"
+"pcg_1358","1358","Other Regulated Provisions","equity","False","Autres provisions réglementées"
+"pcg_1410","1410","Bond Loans","liability_non_current","False","Emprunts obligataires"
+"pcg_1481","1481","Loans from Credit Institutions","liability_non_current","False","Emprunts auprès des établissements de crédit"
+"pcg_1482","1482","Advances from the State","liability_non_current","False","Avances de l'État"
+"pcg_1483","1483","Liabilities Related to Equity Investments","liability_non_current","False","Dettes rattachées à des participations"
+"pcg_1484","1484","Fund Notes","liability_non_current","False","Billets de fonds"
+"pcg_1485","1485","Advances Received and Blocked Current Accounts","liability_non_current","False","Avances reçues et comptes courants bloqués"
+"pcg_1486","1486","Suppliers of Non-Current Assets","liability_non_current","False","Fournisseurs d'immobilisations"
+"pcg_1487","1487","Deposits and Guarantees Received","liability_non_current","False","Dépôts et cautionnements reçus"
+"pcg_1488","1488","Miscellaneous Financial Liabilities","liability_non_current","False","Dettes de financement diverses"
+"pcg_1511","1511","Provisions for Disputes","liability_non_current","False","Provisions pour litiges"
+"pcg_1512","1512","Provisions for Warranties Given to Customers","liability_non_current","False","Provisions pour garanties données aux clients"
+"pcg_1513","1513","Provisions for Own Insurer","liability_non_current","False","Provisions pour propre assureur"
+"pcg_1514","1514","Provisions for Losses on Futures Markets","liability_non_current","False","Provisions pour pertes sur marchés à terme"
+"pcg_1515","1515","Provisions for Fines, Double Duties, Penalties","liability_non_current","False","Provisions pour amendes, doubles droits, pénalités"
+"pcg_1516","1516","Provisions for Foreign Exchange Losses","liability_non_current","False","Provisions pour pertes de change"
+"pcg_1518","1518","Other Provisions for Risks","liability_non_current","False","Autres provisions pour risques"
+"pcg_1551","1551","Provisions for Taxes","liability_non_current","False","Provisions pour impôts"
+"pcg_1552","1552","Provisions for Pensions and Similar Obligations","liability_non_current","False","Provisions pour pensions de retraite et obligations similaires"
+"pcg_1555","1555","Provisions for Deferred Expenses","liability_non_current","False","Provisions pour charges à répartir sur plusieurs exercices"
+"pcg_1558","1558","Other Provisions for Expenses","liability_non_current","False","Autres provisions pour charges"
+"pcg_1601","1601","Headquarters Liaison Accounts","off_balance","False","Comptes de liaison du siège"
+"pcg_1605","1605","Establishments' Liaison Accounts","off_balance","False","Comptes de liaison des établissements"
```

Comment:
I have to say that it is really strange to have two and only two off-balance accounts

Reply:
Yes, that doesn't make sense. It could either be an current asset or liability, depending on the balance. I'll put it as a current liability, since it is between liability accounts.

Reply:
Just thinking that I should include them somewhere in the balance sheet then, although they should be empty by the end of the year.

Reply:
Okay, I read how liaison accounts are used in Belgium e.g., and what they say:

- The Headquarters Liaison Accounts is basically what subsidiaries still owe to the headquarters, so a liability from their point
- The Establishments' Liaison Accounts is like a "compte courant" from the HQ to the subsidiaries, so an asset from their point

I guess it makes sense to define them as such. I'll also integrate them in the BS.

Reply:
> I'll also integrate them in the BS.

Ping me there when it's done :wink:

Reply:
@william-andre The accounts have been set to the correct type and added to the balance sheet (with `hide_if_zero`, since normally they should be empty at the end of the year and not appear on the BS), together with the translations.